packages/api/cms-api/src/kubernetes/kubernetes.module.ts

```diff
@@ -3,9 +3,16 @@ import { DynamicModule, Global, Module } from "@nestjs/common";
 import { KUBERNETES_CONFIG } from "./kubernetes.constants";
 import { KubernetesService } from "./kubernetes.service";
 
-export interface KubernetesConfig {
+export type ClusterKubernetesConfig = {
     helmRelease: string;
-}
+};
+
+export type LocalKubernetesConfig = {
+    namespace: string;
+    helmRelease: string;
+};
+
+export type KubernetesConfig = ClusterKubernetesConfig | LocalKubernetesConfig;
```

Comment:
I've considered a more explicit approach using "mode" as discriminator but decided against it as it would require additional effort to make it backwards compatible and I think it is not necessary to be explicit here.